pkg/common/util/v1/testutil/util_test.go

```diff
@@ -12,16 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tensorflow
+package testutil
 
 import (
 	"testing"
 
+	tfv1 "github.com/kubeflow/tf-operator/pkg/apis/tensorflow/v1"
```

Comment:
no need to move this file? This is util for tensorflow job only. Other folders still won't leverage it even it's moved to `pkg/common` ?

Reply:
can easily delete the file but if you look at what the actual test is testing, it's just testing the helper functions in testutil/util.go. Happy to just delete though if we don't want testutil to have tests.